src/rules/selector-pseudo-class-whitelist/__tests__/index.js

```diff
@@ -0,0 +1,106 @@
+import {
+  messages,
+  ruleName,
+} from ".."
+import rules from "../../../rules"
+import { testRule } from "../../../testUtils"
+
+const rule = rules[ruleName]
+
+testRule(rule, {
+  ruleName,
+  config: [[
+    "hover",
+    "nth-child",
+    "root",
+    "placeholder",
+  ]],
+  skipBasicChecks: true,
```

Comment:
Curious: why are you skipping the basic checks on all of these? Is there a pseudo-class in them?